addon/helpers/list.js

```diff
@@ -1,6 +1,16 @@
 import { MicroState } from 'ember-microstates';
+import assign from '../utils/assign';
 
 export default MicroState.extend({
+
+  wrap(value) {
+    return assign(value, {
```

Comment:
Looking at this, it strikes me that we'd be better served by using `defineProperty` here so that `valueOf` would not be enumerable, which it is when set via `assign`